jib-gradle-plugin/README.md

```diff
@@ -74,32 +83,59 @@ jib.to.image = 'my-docker-id/my-app'
 Build your container image with:
 
 ```shell
-gradle build jib
+gradle jib
 ```
 
 Subsequent builds are much faster than the initial build. 
 
-If you want to clear Jib's build cache and force it to re-pull the base image and re-build the application layers, run:
+*Having trouble? Let us know by [submitting an issue](/../../issues/new), contacting us on [Gitter](https://gitter.im/google/jib), or posting to the [Jib users forum](https://groups.google.com/forum/#!forum/jib-users).*
+
+#### Build to Docker daemon
+
+Jib can also build your image directly to a Docker daemon. This uses the `docker` command line tool and requires that you have `docker` available on your `PATH`.
 
 ```shell
-gradle clean build jib
+gradle jibDockerBuild
 ```
 
-*Having trouble? Let us know by [submitting an issue](/../../issues/new), contacting us on [Gitter](https://gitter.im/google/jib), or posting to the [Jib users forum](https://groups.google.com/forum/#!forum/jib-users).*
+##### Minikube Docker daemon
+
+If you are using [`minikube`](https://github.com/kubernetes/minikube)'s remote Docker daemon, make sure you [set up the correct environment variables](https://github.com/kubernetes/minikube/blob/master/docs/reusing_the_docker_daemon.md) to point to the remote daemon:
+
+```shell
+eval $(minikube docker-env)
+gradle jibDockerBuild
+```
 
 ### Run `jib` with each build
 
 You can also have `jib` run with each build by attaching it to the `build` task:
 
 ```groovy
-tasks.build.finalizedBy tasks.jib
+tasks.build.dependsOn tasks.jib
 ```
 
 Then, ```gradle build``` will build and containerize your application.
 
 ### Export to a Docker context
 
-*Not yet supported*
+Jib can also export to a Docker context so that you can build with Docker, if needed:
```

Comment:
"export a Docker context" instead of "export to a Docker context"? The wording here sounds a bit here like you're exporting the plugin itself.